configuration/analyzer_config.json

```diff
@@ -255,6 +255,17 @@
     "soft_time_limit": 30,
     "queue": "local"
   },
+  "FireHol_IPList": {
+    "type": "observable",
+    "observable_supported": ["ip"],
+    "external_service": true,
+    "description": "check if an IP is in FireHol's IPList",
+    "python_module": "firehol_iplist.FireHol_IPList",
+    "additional_config_params": {
+      "list_names": ["firehol_level1.netset", "cleantalk_1d.ipset"]
```

Comment:
was cleantalk added randomly or there is a specific reason? I guess you used it for development purpose but why keep this here. Maybe we should think about which kind of lists we should provide by default.

Reply:
Thank you for pointing out! I was just writing a comment asking about default set of list names. I should have done it beforehand, my bad. So, what kind of lists should we provide by default?

Reply:
By default, only Firehol_level1 should be enough. It's composed of many different iplists.